runtime/src/accounts_db.rs

```diff
@@ -419,6 +419,10 @@ pub struct AccountsDB {
     /// Keeps tracks of index into AppendVec on a per slot basis
     pub accounts_index: AccountsIndex<AccountInfo>,
 
+    pub uncleaned_keys: Mutex<HashMap<Slot, Vec<Pubkey>>>,
+
+    pub zero_lamport_accounts: RwLock<HashSet<Pubkey>>,
```

Comment:
I think converting these to `DashMap` and `DashSet` respectively may not be a bad idea, @ryoqun was seeing a lot of contention on locks between replay/ and the `self.thread_pool_clean.install` in `clean`

Reply:
Yea, but the critical sections of these is much smaller and it's only in the case of 0-lamport stores/updates which is much rarer than the generic case of taking locks on the index.

Reply:
ok, used dashset/dashmap